lib/Launcher.js

```diff
@@ -231,10 +198,32 @@ class Launcher {
   }
 
   /**
+   * @param {!ChromeArgOptions=} options
    * @return {!Array<string>}
    */
-  static defaultArgs() {
-    return DEFAULT_ARGS.concat(AUTOMATION_ARGS);
+  static defaultArgs(options = {}) {
+    const userArgs = options.args || [];
```

Comment:
we had `Array.isArray` check before for user args

Reply:
If someone passes a truthy but non array value, we should throw an error instead of silently moving on.

Reply:
I'm all in for an assert